Examples/Supporting Files/TestData.swift

```diff
@@ -50,3 +50,33 @@ let testImages: [UIImage] = [
   UIImage(named: "5")!,
   UIImage(named: "6")!
 ]
+
+let testArticles =
```

Comment:
nit, think this format looks better

```
let testArticles = [
   ArticleData...
]
```

Reply:
What do you think about this idea.

We use color and place holder text to populate the cells instead of hardcoding fake data.
This way we can generate a lots of cells. 

We can use something like this:
```swift
let color = UIColor(hue: CGFloat(index) / CGFloat(cellCount), saturation: 0.68, brightness: 0.98, alpha: 1)
let data = ArticleData(color: color,
                    title: "Article \(index)",
                    subTitle:"This is the subtitle for article \(index)")
view.populateWithArticle(data)
```

Reply:
nice, let me try

Reply:
Done